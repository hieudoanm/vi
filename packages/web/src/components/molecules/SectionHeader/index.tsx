import React from 'react';

const SectionHeader: React.FC = ({ children }) => {
  return (
    <div className="pb-8">
      <h2 className="uppercase text-2xl">{children}</h2>
    </div>
  );
};

export default SectionHeader;
