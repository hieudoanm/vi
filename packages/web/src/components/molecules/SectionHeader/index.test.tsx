import SectionHeader from '.';
import { render } from '@testing-library/react';

describe('SectionHeader', () => {
  test('render default', () => {
    const wrapper = render(<SectionHeader />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
