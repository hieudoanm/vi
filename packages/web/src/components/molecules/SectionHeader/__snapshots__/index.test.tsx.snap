// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionHeader render default 1`] = `
<div>
  <div
    class="pb-8"
  >
    <h2
      class="uppercase text-2xl"
    />
  </div>
</div>
`;
